### DISCUSSION TITLE
Add a counter of how many successful seeds have been found

### DISCUSSION THREAD

**Discussion by RavingPlatypi** [Zodsmar/SeedSearcherStandaloneTool#131] [Ideas]
It would be useful to know how many seeds the searcher has found and how many it still needs to find. That way I can approximate how much time I need to keep the searcher on for, you know?

Pretty minor feature, so if it is a pain then don't worry about it. But it seems pretty easy to implement with a simple counter. Thanks :)

**Comment by Zodsmar** [comment]
What do you mean how many it needs to find? The searcher will go till either it has found the limit your looking for or until it has searched all possible seeds. (which would take a very very long time)

**Reply by RavingPlatypi** [reply]
Apologies for being unclear. I meant that I would like to see a counter of how many seeds it has already found. For example, the tool would display something like "Seeds Found: 400/10000."

**Reply by Zodsmar** [reply]
Ah, I see. Yeah something like that I can add. Ill add it to the todo.

**Reply by RavingPlatypi** [reply]
Amazing! Thanks a lot :)
